packages/SwingSet/test/workers/vat-target.js

```diff
@@ -63,9 +63,16 @@ export function buildRootObject(vatPowers, vatParameters) {
 
   // crank 5: dispatch.notify(pF, false, ['data', callbackObj])
 
+  const contents = [];
```

Comment:
Is this leftover? I don't see any changes in the rest of the test that might call it.

Reply:
The `contents` is for the `append(thing)` method we added.